src/otx/v2/configs/detection/otx_mmdet_default.yaml

```diff
@@ -6,7 +6,7 @@ data:
   train_type: Incremental
 
 model:
-  name: otx_mobilenetv2_atss
+  name: otx_det_mobilenetv2_atss
```

Comment:
I prefer the previous name since ATSS could represent the task. Could you explain why you changed the name? Maybe there is no issue of the duplication.

Reply:
Same goes on the `otx_iseg_resnet50_maskrcnn`

Reply:
It's because to specify model task by name. Since iseg models and detection models share MODEL_CONFIGS, there is no way to filter only detection(or iseg) model by pattern. With this kind of naming, we can call list_models function like this
`list_model("otx_det*")`

Reply:
Ah okay, I thought that OTX2.0 could identify each models by using the folder architecture.